cocos2d/core/components/CCWebView.js

```diff
@@ -175,11 +176,13 @@ var WebView = cc.Class({
 
     /**
      * !#en
-     * Set javascript interface scheme.
-     * Note: only available on Android and iOS at the moment.
+     * Set javascript interface scheme (see also setOnJSCallback). <br/>
+     * Note: Only support Android and iOS, Web end usage, please go to the official document view.<br/>
```

Comment:
Note: Supports only on the Android and iOS. For HTML5, please refer to the official documentation.